packages/Button/Button.vue

```diff
@@ -88,8 +90,8 @@ i {
 }
 
 .btn--primary {
-  color: var(--White);
-  background-color: var(--primary);
+  color: $white;
+  background-color: $primary;
 }
 .btn--primary:hover {
   background-color: var(--secondary);
```

Comment:
You shouldn't have any `var()` nowhere in the codebase, that's the whole purpose of moving to `scss`. I've seen a few in many files that were not here before the resolution of conflicts.

I don't think the resolution of conflicts was done correctly...

Reply:
Fixed